src/pages/workspace/card/WorkspaceCardVBAWithECardView.js

```diff
@@ -32,6 +32,13 @@ const WorkspaceCardVBAWithECardView = props => (
                 shouldShowRightIcon: true,
                 iconRight: Expensicons.NewWindow,
             },
+            {
+                title: props.translate('workspace.common.settlementFrequency'),
```

Comment:
This needs to be behind the `monthlySettlement` beta

Reply:
Have been researching and found an SO but all the info is about how to implement Beta in Web-Expensify
You know how i can do same in newdot?

Reply:
I'm not sure myself, I don't really work in newDot. But if you search the codebase you should be able to find some examples. Alternatively you can ask in #eng-chat

Reply:
check `Permissions.canUseWallet(props.betas)` and you can do something similar